WORKSPACE

```diff
@@ -18,6 +18,9 @@ workspace(name = "org_tensorflow")
 load("//tensorflow:workspace.bzl", "tf_workspace")
 tf_workspace()
 
+load("//third_party/gpus:cuda_configure.bzl", "cuda_configure")
```

Comment:
Shouldn't this be at the begining of the workspace file?

Maybe you actually want to add the cuda_configure to the tf_workspace macros for enabling it for ppl depending on it (like serving).